pkg/seeds/seeds.go

```diff
@@ -36,32 +39,35 @@ func SeedRepositoryConfigurations(db *gorm.DB, size int, options SeedOptions) er
 	return nil
 }
 
-func createOrgId(existingOrgId string, index int) string {
+func createOrgId(existingOrgId string) string {
 	orgId := "4234"
 	if existingOrgId != "" {
 		orgId = existingOrgId
 	} else {
-		// Only add random numbers if no specific existingOrgId is populated
-		if index < 15 || index > 915 {
+		randomNum := rand.Intn(5)
+		if randomNum == 3 {
 			orgId = strconv.Itoa(rand.Intn(9999))
 		}
 	}
 	return orgId
 }
 
-func createVersionArray(index int) []string {
-	versionArray := []string{}
+func createVersionArray(versionArray []string, emptyArray bool) []string {
+	if len(versionArray) > 0 || emptyArray {
+		return versionArray
+	}
+	randomNum := rand.Intn(20)
 	length := 0
```

Comment:
```suggestion
	length := rand.Intn(4)
```
Could this replace all the if statements? I think currently it is most likely to produce versionArray=["7"], but is the predictability still a factor here?

Reply:
And I could see similar changes to arch and OrgID generation to remove some of the conditional logic